pxr/imaging/plugin/hdRpr/rprApi.cpp

```diff
@@ -1690,9 +1690,7 @@ class HdRprApiImpl {
 
             if (m_state == kStateRender && config->IsDirty(HdRprConfig::DirtyRenderQuality)) {
                 TfToken activeRenderQuality;
-                if (m_rprContextMetadata.pluginType == kPluginTahoe) {
-                    activeRenderQuality = HdRprCoreRenderQualityTokens->Full;
```

Comment:
After removing "Full" setting from generateRenderSettingFiles.py token HdRprCoreRenderQualityTokens->Full is unavailable